ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -77,6 +77,16 @@
                   class="group-state-nav-icon"
                   title="Collapse job groups">-</span>
           )</span>
+
+          <!-- Toggle Duplicate Jobs -->
+          <span class="btn btn-view-nav btn-sm btn-hide-duplicate-jobs"
+                tabindex="1" role="button" 
+                title="{{ showDuplicateJobs ? 'Hide Duplicate Jobs' : 'Show Duplicate Jobs' }}"
+                ng-click="groupState !== 'expanded' && toggleShowDuplicateJobs()"
+                ng-disabled="groupState === 'expanded'">
+            <i class="fa"
+               ng-class="{ 'fa-eye-slash': !showDuplicateJobs,
```

Comment:
Let's actually go with just text, rather than the eye.  We already use the eye for "Excluded Jobs" anyway.  What about just "Show Dupes" and "Hide Dupes"?